Intake/Resources/Localizable.xcstrings

```diff
@@ -16,7 +16,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "You may login to your existing account. Or create a new one if you don't have one already."
+            "value" : "You may login to your existing account. Or create a new one if you don't have one already. You will have to enter your details of name, date of birth, and your preferred email."
```

Comment:
This reads like date of birth will be required? You configured it as optional above, though. I would propose to either update the configuration above or update the wording that it is explicit what personal details are required to be entered.